packages/catalog-realm/catalog-app/components/grid.gts

```diff
@@ -54,69 +56,86 @@ export class CardsGrid extends GlimmerComponent<CardsGridSignature> {
   };
 
   <template>
-    <ul
-      class='cards {{@selectedView}}-view'
-      data-test-cards-grid-cards
-      ...attributes
-    >
-      {{#let
-        (component @context.prerenderedCardSearchComponent)
-        as |PrerenderedCardSearch|
-      }}
-        <PrerenderedCardSearch
-          @query={{@query}}
-          @format='fitted'
-          @realms={{@realms}}
-          @isLive={{true}}
-        >
-          <:loading>
-            Loading...
-          </:loading>
-          <:response as |cards|>
-            {{#each cards key='url' as |card|}}
-              <li
-                class='{{@selectedView}}-view-container'
-                data-test-card-url={{card.url}}
-              >
-                {{#if
-                  (and (this.isHydrated card.url) this.cardResource.isLoaded)
-                }}
-                  {{#if this.cardResource.card}}
-                    {{#let
-                      (getComponent this.cardResource.card)
-                      as |Component|
-                    }}
-                      <CardContainer
-                        class='card'
-                        @displayBoundaries={{true}}
-                        data-test-cards-grid-item={{removeFileExtension
-                          card.url
-                        }}
-                        data-cards-grid-item={{removeFileExtension card.url}}
-                        {{on 'click' (fn this.viewCard card)}}
-                      >
-                        <Component />
-                      </CardContainer>
-                    {{/let}}
+    {{#let
+      (component @context.prerenderedCardSearchComponent)
+      as |PrerenderedCardSearch|
+    }}
+      <PrerenderedCardSearch
```

Comment:
I created a `CardList` component in Base realm (could not move to boxel-ui because of type issues), so we can use that instead of re-rewriting this component again. It does need improvements, though. Currently it's only used for the index cards-grid. Do you think you can use it here?